src/esp/bindings_js/modules/vr_utils.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Is there any code redundancy between this file and the existing vr_demo.js?

Reply:
Yeah, see here: https://github.com/facebookresearch/habitat-sim/pull/1364#discussion_r664908573

The existing vr_demo.js will be deleted in https://github.com/facebookresearch/habitat-sim/pull/1366, so this redundancy will be resolved as soon as that PR gets merged.